cmd/audiusd/main.go

```diff
@@ -27,7 +27,7 @@ import (
 
 func main() {
 	noTLS := flag.Bool("no-tls", false, "Disable TLS and use only port 80")
-	noMediorum := flag.Bool("no-mediorum", false, "Disable Mediorum i.e. core node only")
+	noMediorum := flag.Bool("mediorum", false, "Enable Mediorum")
```

Comment:
We don't have to make this decision now, but it might be better to assume node operators don't know what "mediorum" is and they'd rather the flag be something like `--enable-storage` etc.

Again, not a decision we have to make right now.